src/lando/ui/legacy/forms.py

```diff
@@ -63,39 +63,41 @@ def clean(self):
 class UpliftRequestForm(UpliftAssessmentForm):
     """Form used to request uplift of a stack."""
 
-    source_revision_id = forms.RegexField(
-        regex="^D[0-9]+$",
-        widget=forms.widgets.HiddenInput,
-        required=False,
+    source_revision_ids = forms.ModelMultipleChoiceField(
```

Comment:
Should this be named `source_revisions`?

Reply:
I called it `source_revision_ids` as we're using `to_field_name="revision_id"` below. The revision IDs are what is submitted in the form, as opposed to the PK of the corresponding `Revision` model. You can see this in the tests for the uplift request creation view.

Reply:
I think this can still cause some confusion, since `cleaned_data["source_revision_ids"]` implies that it should contain integers, not Revision objects. I think the convention is to drop the `id` in the form, if the expected data is a related model.

>The revision IDs are what is submitted in the form, as opposed to the PK of the corresponding Revision model

Not sure this would be relevant to the clean method or the name of the field. You could pick any field that shows in the selection, but ultimately we care about the actual referenced objects, no?

Reply:
Renamed this field to `source_revisions`.